projects/v2/src/app/material.module.ts

```diff
@@ -1,55 +1,55 @@
 import { NgModule } from '@angular/core';
```

Comment:
I don't see this file being used anywhere. Delete it if it isn't being used!

Reply:
Done